packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -27,12 +99,39 @@ export const OAuthLoginPage = () => {
   }, [redirect_uri])
   const redirectUriHost =
     parsedRedirectUri == null ? null : parsedRedirectUri.host
-  const { token } = queryString.parse(hash)
   const account = useSelector(getAccountUser)
   const isLoggedIn = Boolean(account)
-  const [email, setEmail] = useState('')
-  const [password, setPassword] = useState('')
-  const [submitError, setSubmitError] = useState<string | null>(null)
+  const [userEmail, setUserEmail] = useState<string | null>(null)
+  useEffect(() => {
+    const getAndSetEmail = async () => {
+      let email: string
+      try {
+        email = await AudiusBackend.getUserEmail()
+      } catch {
+        setUserEmail(null)
+        window.location.href = ERROR_PAGE
```

Comment:
not sure what the standard approach is here, but thinking we should ideally use `react-router` to push to error_page?